_includes/v22.2/start-in-docker/mac-linux-steps-single-node.md

```diff
@@ -0,0 +1,128 @@
+  - `COCKROACH_DATABASE`
+  - `COCKROACH_USER`
+  - `COCKROACH_PASSWORD`
+
+  To prevent data loss, the environment variables are ignored if the `/cockroach/cockroach-data` directory within the container is not empty. To re-initialize the default database, user, and password from the environment variable values, delete the contents of `/cockroach/cockroach-data` within the running container.
+
+- You can optionally mount a directory of initialization scripts into the `docker-entrypoint-initdb.d` directory within the container. These scripts are run after CockroachDB starts and after the database and user (if specified) have been created. The scripts run in the alphanumeric sort order imposed by your locale.
+
+This section shows how to start a single-node cluster that uses these features.
+
+### Step 1. Create Docker volumes for the node
+
+Cockroach Labs recommends that you store cluster data in a Docker volume rather than in the storage layer of the running container. Otherwise, if a Docker container is inadvertently deleted, its data is inaccessible.
+
+To create the [Docker volume](https://docs.docker.com/storage/volumes/) where the cluster will store its data:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+docker volume create roach-single
+~~~
+
+
+
+### Step 2. Start the cluster
+
+1. Start the cluster node.
+
+    The following command starts a single-node cluster that:
+ 
+    - Stores its data in the `roach-single` volume on the Docker host, which is mounted on the `/cockroach/cockroach-data` directory within the container.
+    - If the `/cockroach/cockroach-data` within the container is empty, the specified database, user, and password are created automatically. Instead of specifying each value directly by using the `-e` or `--env` flag, you can store them in a file on the Docker host. Use one key-value pair per line and set the `--env-file` flag to the file's path.
+    - Bind-mounts the `~/init-scripts` directory on the Docker host onto the `/docker-entrypoint-initdb.d` directory within the container. Initialization scripts stored in this directory are run after CockroachDB starts and the default database, user, and password are initialized.
+    - Accepts database client connections on IP address 172.18.0.3 (which resolves to the hostname `roach-single` within the Docker container) on port 26257.
+    - Accepts connections to the DB Console on IP address 172.18.0.3 (which resolves to the hostname `roach-single` within the Docker container) on port 8080.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    docker run -d \
+              --env COCKROACH_DATABASE={DATABASE_NAME} \
+              --env COCKROACH_USER={USER_NAME} \
+              --env COCKROACH_PASSWORD={PASSWORD} \
+              --name=roach-single \
+              --hostname=roach-single \
+              --ip=172.18.0.3 \
+              --advertise-addr=172.18.0.3 \
```

Comment:
There's no `--advertise-addr` flag for docker run, so I think we should remove it.

Reply:
Done.